test/govspeak_test.rb

```diff
@@ -891,4 +891,26 @@ class GovspeakTest < Minitest::Test
       assert_html_output(expected_html)
     end
   end
+
+  test "Figure" do
+    govspeak = "$Figure
+$Alt
+Example of the part-whole model using the number 28. The number 20 has already been added to the lower left field, the lower right field is blank ready to be filled in.
+$EndAlt
+$URLhttps://www.early-career-framework.education.gov.uk/teachfirst/wp-content/uploads/sites/4/2020/08/Part-whole-model.jpg$EndURL
+$CaptionFigure 1: Part-whole model.
+$EndCaption
```

Comment:
I would add some whitespace here to make it easier to read. `$CaptionFigure 1:` is not really easy to understand ;)